portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="slide.css">
+    <script src="slide.js" defer></script>
+  </head>
+  <body>
+    <div id="content">
+      <div class="main">
+        <h1>My Image Gallary</h1>
+        <div class="slides" id="slide1">
+          <img class="pic pic-hide" src="images/cat0.jpg" alt="My Cat At His 4th Month">
```

Comment:
Formatting for all alt tags. Should be sentence-like. "My cat at his 4th month."

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback